### ISSUE TITLE
CELA P1_A11y_code.visualstudio.com_WCAG2.1_Homepage_MagicReader: The textual content isn't visible as the spacing between the text increases after we check with the magic reader tool.

### ISSUE THREAD

**Issue by khansahab15** [microsoft/vscode-docs#3433]
“Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11ySev2; #A11yTCS; #A11yWCAG2.1; #Benchmark; #CELA P1; #Chrome; #MagicReader; #DesktopWeb; #MAS1.4.12; #Win10; #Visual Studio Code; #CELA P1_A11y_code.visualstudio.com_WCAG2.1

**Environment Details:** 
Application Name:  code.visualstudio.com
#URL:https://code.visualstudio.com/docs/editor/versioncontrol
Windows Version: Windows10
Browser version: Chrome version 79.0.3945.130 (Official Build) (64-bit)
Tool: Magic Reader.

**Repro Steps:** 
1.	Launch the application URL: https://code.visualstudio.com/docs/editor/versioncontrol
2.	Home screen will open
3.	Navigate to the “GitHub pull requests” extension below “Working with pull requests”
4.	Check if there is proper spacing using magic reader tool.

**Actual:**
The textual content isn't visible as the spacing between the text increases after we check with the magic reader tool.

**Expected:**
After checking  the text spacing with magic reader the text should be clearly visible with appropriate spacing and without overlapping.

**User Impact:**
Low vision users might not be able to view the content easily that is present on the screen.

**Attachment:**
[#3433.zip](https://github.com/microsoft/vscode-docs/files/4265430/3433.zip)

**Comment by gregvanl** [comment]
Hi @khansahab15, can you provide information on the magic reader tool? What does it do and how can try test our site with it. Thanks

**Comment by khansahab15** [comment]
Hey @gregvanl 
Magic reader is an google chrome extension used to check the Text spacing with the intent that no loss of content or functionality occurs.

To get magic reader just search for it in the google chrome Extension store and ADD it to the browser.

**Comment by gregvanl** [comment]
Found the Chrome extension https://chrome.google.com/webstore/detail/magic-reader/dgccjjilmaehpmdmabjhhfibbmlaflfd
You can see this issue if you set Bigger Line Spacing through the Options Test.

**Comment by gregvanl** [comment]
Before:
![image](https://user-images.githubusercontent.com/12818376/77085659-e16d6600-69bd-11ea-9047-4badc43d9ae1.png)
With Magic Reader on:
![Uploading image.png…]()

**Comment by gregvanl** [comment]
The text that is missing is optional description text for the extensions, which is already truncated to 40 characters. The tile still shows the extension name and publisher and when selected takes the user to the Marketplace, where they can review the full extension description and README. 
Closing as this does not block functionality.

**Comment by 12-shweta** [comment]
#CELA_code.visualstudio.com_WCAG2.1_FEB2020;
